README.md

```diff
@@ -170,6 +171,13 @@ Behind the scenes, the extension builds the Pypi dependencies tree by running `p
 1. Open a new terminal and activate your Virtualenv as instructed [here](https://virtualenv.pypa.io/en/latest/userguide/#activate-script).
 1. Install your python project and dependencies according to your project specifications.
 
+## NuGet Projects
+Behind the scenes, the extension builds the NuGet dependencies tree using [NuGet deps tree](https://github.com/JFrog/nuget-deps-tree).
+
+Important:
+To have your project dependencies scanned by JFrog Xray, make sure the `nuget` or `dotnet` CLI are installed on your local machine and that they are in your system PATH.
```

Comment:
Till we're sure the .NET Core CLI is supported, let's remove it from here. Same goes for the following sentence.